app/representers/cms/page_representer.rb

```diff
@@ -1,32 +1,28 @@
+# frozen_string_literal: true
+
 module CMS::PageRepresenter
   include ThreeScale::JSONRepresenter
 
-  wraps_resource
+  wraps_resource ->(*) { self.class.data_tag }
 
   property :id
-
-  with_options(if: ->(options) { options[:short] == false }) do |p|
-    p.property :draft, render_nil: true
-    p.property :published, render_nil: true
-  end
-
+  property :created_at
+  property :updated_at
+  property :title
   property :system_name
```

Comment:
For some reason, `system_name` doesn't appear when it's empty, it should appear as `""`.

After https://github.com/3scale/porta/pull/3163, `system_name` is not going to be empty anymore. But I think this should be fixed anyway, it's still not clear what are we going to do with that PR.